### PR TITLE
ML glossary: updated precision and recall definitions

### PR DESCRIPTION
**Pull request by pkulikov**
Also updated wording in F-score definition to be consistent with wording of other evaluation metrics: Root of mean squared error, for example.

### CODE REVIEW

File: docs/machine-learning/resources/glossary.md [hunk 1]
```diff
@@ -75,19 +75,19 @@ A feature extraction scheme for text data. Any sequence of N words turns into a
...
+A feature vector consisting only of numerical values. This is similar to `double[]`.
 
 ## Pipeline
 
 All of the operations needed to fit a model to a dataset. A pipeline consists of data import, transformation, featurization, and learning steps. Once a pipeline is trained, it turns into a model.
 
 ## Precision
 
-The proportion of true results to positive results.
+In [classification](#classification), the precision for a class is the number of items correctly predicted as belonging to that class divided by the total number of items predicted as belonging to the class.
```
Comment (pkulikov) [inline docs/machine-learning/resources/glossary.md]: Or should I reference binary classification, as I see precision and recall only in [BinaryClassificationMetrics class](https://docs.microsoft.com/en-us/dotnet/api/microsoft.ml.models.binaryclassificationmetrics?view=ml-dotnet)?

**Review by aditidugar-zz — COMMENTED** [review]
LGTM.

**Review by mairaw — APPROVED** [review]
Approving this given @aditidugar's review. @pkulikov can you just make a change to the ms.date before we merge?

**Comment by pkulikov** [comment]
@mairaw Done!

**Pull request merged**